spec/features/membership_confirmation_spec.rb

```diff
@@ -0,0 +1,107 @@
+RSpec.describe "membership confirmation process" do
+  let(:member) {
+    Member.create!(
+      full_name: "John Doe",
+      email: "john.doe@example.com",
+      address: "22 Pollen Street, Grey Lynn, Auckland 1021",
+    )
+  }
+
+  context "via email" do
+    it "allows them to confirm membership" do
+      MembershipMailer.confirm(member).deliver_now
+
+      email = ActionMailer::Base.deliveries.last
+
+      expect(email.to).to include "john.doe@example.com"
+
+      expect(email.body.encoded).to match(
+        %r{
+          (?-x:Hi John Doe).*
+          (?-x:Membership of the Ruby New Zealand Society has grown in recent).*
+          (?-x:years, and we \(the current Ruby New Zealand Committee\) want to).*
+          (?-x:ensure that we have an accurate list of current members.).*
+          (?-x:If you would like to remain a member of Ruby New Zealand).*
+          (?-x:please confirm your existing membership here:).*
+          (?-x:Being a member of Ruby New Zealand allows you to have your say).*
+          (?-x:about the committee and other society matters at the annual AGM).*
+          (?-x:Kind regards.*The 2021 New Zealand Ruby Committee.)
+        }xm
+      )
+
+      url = URI.extract(email.body.encoded).detect do |url|
+        url.match(%r[example.com])
+      end
```

Comment:
Very smart of you :+1:

Reply:
Haha, thanks! I can't claim credit 😉  I got the idea from the members feature spec you added six months ago! ✨

Reply:
> Haha, thanks! I can't claim credit wink I got the idea from the members feature spec you added six months ago! sparkles

haha, I did not remember writing that code :joy:.